assets/helpers/csrf/csrfReducer.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+// ----- Imports ----- //
+
+import type { Action } from './csrfActions';
+
+
+// ----- Types ----- //
+
+type Csrf = {
+    token: ?string,
+};
+
+
+// ----- Setup ----- //
+
+const initialState: Csrf = window.guardian && window.guardian.csrf ? {
```

Comment:
This is entirely based on the user reducer.  Should we fix them both at the same time?